### PR TITLE
Fix: crab UI show wrong PnL 

### PR DESCRIPTION
**Pull request by KMKoushik**
# Task:
Crab shows bloated values

Fixes # (issue)

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested
Checked production values

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

### CODE REVIEW

File: packages/frontend/src/hooks/useCrabPosition.ts [hunk 1]
```diff
@@ -8,14 +8,14 @@ import { useAtomValue } from 'jotai'
 import { useTokenBalance } from './contracts/useTokenBalance'
 import { addressesAtom } from 'src/state/positions/atoms'
 import { indexAtom } from 'src/state/controller/atoms'
+import useAppCallback from './useAppCallback'
```
Comment (alexisgauba) [inline packages/frontend/src/hooks/useCrabPosition.ts:11]: what is useAppCallback? / how does it differ from normal useCallback?

**Review by alexisgauba — COMMENTED** [review]
Been long enough since I've really gone deep into the code to have a good sense on the implementation here 

Looks ok to me when I test in prod, but what were the key issues and how did you solve them? 

Also is it worth adding a test for the fix per Anil's suggestion or does it only make sense to start doing that after the testing PR is merged?

File: packages/frontend/src/hooks/useCrabPosition.ts (hunk 1 above)
Comment (KMKoushik) [reply packages/frontend/src/hooks/useCrabPosition.ts:11]: This is a new hook added by @adam5909099  https://github.com/opynfinance/squeeth-monorepo/pull/272 Helps to do the `.toString` of BigNumbers automatically so we don't have to ignore any warnings thereby missing any important dependency is missed.

Comment (KMKoushik) [reply packages/frontend/src/hooks/useCrabPosition.ts:11]: Also can't have tests until we find out how to test PnLs correctly

**Review by alexisgauba — APPROVED** [review]

**Review by adam5909099 — APPROVED** [review]
Looks great to me!

**Pull request merged**